dev/cmp/tests/vals.rs

```diff
@@ -70,6 +70,7 @@ fn test_libnogg_vals() {
 	cmp_output!("noise-6ch.ogg", 0);
 	cmp_output!("noise-stereo.ogg", 0);
 	cmp_output!("partial-granule-position.ogg", 2);
+    #[cfg(target_os = "linux")]
```

Comment:
Can you use tabs here?

Reply:
It's weird because I can see the line aligned properly if I look at the complete [file](https://github.com/Luni-4/lewton/blob/a49b8999cd95654b7027b6ce6c0ab314298fcf63/dev/cmp/tests/vals.rs#L73)

Reply:
Sorry, I have forgotten that GitHub reformats the source files :)
Done!